test/unit/requestWrapper.test.js

```diff
@@ -273,65 +272,64 @@ describe('sendRequest', () => {
       },
     };
 
-    axios.mockImplementation(requestParams => {
+    mockAxiosInstance.mockImplementation(requestParams => {
       requestParams.paramsSerializer(requestParams.params);
       return Promise.resolve('res');
     });
 
-    sendRequest(parameters, (err, body, res) => {
+    requestWrapperInstance.sendRequest(parameters, (err, body, res) => {
       // assert results
-      expect(axios.mock.calls[0][0].data).toEqual('a=a&b=b');
-      expect(axios.mock.calls[0][0].url).toEqual(
+      expect(mockAxiosInstance.mock.calls[0][0].data).toEqual('a=a&b=b');
+      expect(mockAxiosInstance.mock.calls[0][0].url).toEqual(
         'https://example.ibm.com/v1/environments/environment-id/configurations/configuration-id'
       );
-      expect(axios.mock.calls[0][0].headers).toEqual({
+      expect(mockAxiosInstance.mock.calls[0][0].headers).toEqual({
         'Accept-Encoding': 'compress',
         'test-header': 'override-header-value',
         'add-header': 'add-header-value',
         'Content-type': 'application/x-www-form-urlencoded',
       });
-      expect(axios.mock.calls[0][0].httpsAgent.options.rejectUnauthorized).toEqual(
-        parameters.options.rejectUnauthorized
-      );
-      expect(axios.mock.calls[0][0].method).toEqual(parameters.options.method);
-      expect(axios.mock.calls[0][0].params).toEqual(parameters.options.qs);
-      expect(axios.mock.calls[0][0].responseType).toEqual('json');
+      expect(mockAxiosInstance.mock.calls[0][0].method).toEqual(parameters.options.method);
+      expect(mockAxiosInstance.mock.calls[0][0].params).toEqual(parameters.options.qs);
+      expect(mockAxiosInstance.mock.calls[0][0].responseType).toEqual('json');
       expect(res).toEqual('res');
-      expect(axios.mock.calls.length).toBe(1);
+      expect(mockAxiosInstance.mock.calls.length).toBe(1);
       done();
     });
   });
 
-  it('should keep parameters in options that are not explicity set in requestwrapper', done => {
-    const parameters = {
-      defaultOptions: {
-        body: 'post=body',
-        formData: '',
-        qs: {},
-        method: 'POST',
-        rejectUnauthorized: true,
-        url:
-          'https://example.ibm.com/v1/environments/environment-id/configurations/configuration-id',
-        headers: {
-          'test-header': 'test-header-value',
-        },
-        responseType: 'buffer',
-      },
-      options: {
-        otherParam: 500,
-      },
-    };
-
-    axios.mockResolvedValue('res');
-
-    sendRequest(parameters, (err, body, res) => {
-      // assert results
-      expect(axios.mock.calls[0][0].otherParam).toEqual(500);
-      expect(res).toEqual('res');
-      expect(axios.mock.calls.length).toBe(1);
-      done();
-    });
-  });
+  // Need to rewrite this to test instantiation with userOptions
```

Comment:
What needs to be done for this? Will you open an issue in the SDK planning board for this so we don't lose it?